app/models/character.rb

```diff
@@ -1,4 +1,5 @@
 class Character < ApplicationRecord
+  has_and_belongs_to_many :users
```

Comment:
Belongs to many users?

Reply:
Ye I created a many-to-many instead of a one-to-many relationship. Changing it now!